src/FieldWarning/Assets/UI/Ingame/BillboardBehavior.cs

```diff
@@ -21,11 +21,17 @@ namespace PFW.UI.Ingame
     public class BillboardBehavior : MonoBehaviour
     {
         [SerializeField]
-        private float ALTITUDE = 10f * Constants.MAP_SCALE;
+        private float ALTITUDE;
         [SerializeField]
         private float SIZE = 0.1f;
 
-        // Update is called once per frame
+        private void OnEnable()
+        {
+            // HACK I don't know why, but setting this on declaration simply wasn't working, so I moved it here.
+            // Nothing else touches this, so I can only assume it is either a Unity bug or a magical network issue.
+            ALTITUDE = 10f * Constants.MAP_SCALE;
```

Comment:
The difference is that the previous version was a default (it can be overriden by editing the inspector field in the prefab) whereas your code overwrites what was set in the inspector.

Reply:
Those changes have now been reversed.